### PR TITLE
XW-658 Replace new top bar with the classic one

### PR DESCRIPTION
**Pull request by rogatty**
https://wikia-inc.atlassian.net/browse/XW-658

/cc @kvas-damian @bkoval 

This is still WIP, we need to adjust styling.

### CODE REVIEW

File: front/styles/main/component/_discussion-hero-unit.scss [hunk 1]
```diff
@@ -39,7 +39,7 @@ $discussion-hero-unit-avatar-dimension: 40px;
 .minimized {
 	&.discussion-hero-unit {
 		padding-bottom: 0;
-		padding-top: 25px;
+		padding-top: 95px;
```
Comment (bkoval) [inline front/styles/main/component/_discussion-hero-unit.scss]: This one https://github.com/Wikia/mercury/pull/1585/files#diff-da50bbcee5d5ccd13251718df8c661e1R8 was changed by 40px but this one changed by 70px. Why is that? :)

**Comment by bkoval** [comment]
:*

File: front/styles/main/component/_discussion-hero-unit.scss (hunk 1 above)
Comment (bkoval) [inline front/styles/main/component/_discussion-hero-unit.scss]: Maybe a variable could make it a bit more readable?

**Comment by kvas-damian** [comment]
@rogatty please take a look

**Comment by rogatty** [comment]
@kvas-damian are you sure you don't need `smartBannerVisible` in `front/templates/main/discussion/post.hbs`?

Apart from that LGTM

**Comment by kvas-damian** [comment]
@rogatty for some reason discussion header is not enabled on post view.
```{{discussion-header
	showContent=false
	headroomEnabled=false
}}```
So I don't think I need it there.

File: front/styles/main/component/_discussion-header.scss [hunk 2]
```diff
@@ -73,3 +76,42 @@ $discussion-header-color-text: #fff;
...
+	margin-top: 3.75rem;
+}
+
+@media #{$desktop-range} {
+	.site-body-discussion {
+		margin-top: 0;
+	}
+}
+
+hr.theme {
```
Comment (bkoval) [inline front/styles/main/component/_discussion-header.scss]: @vforge Do we need such specificity here?

Comment (kvas-damian) [inline front/styles/main/component/_discussion-header.scss]: @bkoval IMO ```theme``` class name should be used at all. It's not descriptive so having more specificity makes sense for me on this case.

Comment (vforge) [inline front/styles/main/component/_discussion-header.scss]: @bkoval My intention was not to change anything else than what's needed (`hr.theme` is from discussions, and `hr` can be universal)

File: front/styles/main/component/_discussion-hero-unit.scss (hunk 1 above)
Comment (vforge) [inline front/styles/main/component/_discussion-hero-unit.scss]: Avatar was not visible - it was under the banner - when it was lower than 90px :) I don't have any idea how to call it and if it's even necessary - we won't re-use it anyway.

File: front/styles/main/component/_discussion-header.scss (hunk 2 above)
Comment (bkoval) [inline front/styles/main/component/_discussion-header.scss]: :+1: Thanks!

**Comment by bkoval** [comment]
LGTM :+1:  :santa: :fire:

**Comment by rafalkalinski** [comment]
LGTM

**Comment by bkoval** [comment]
LGTM

**Comment by bkoval** [comment]
LGTM

**Comment by kvas-damian** [comment]
LGTM

**Pull request merged**